extensions/amp-image-slider/1.0/component.js

```diff
@@ -8,30 +8,33 @@ import {
   useCallback,
   useEffect,
   useImperativeHandle,
-  useLayoutEffect,
-  useMemo,
   useRef,
   useState,
 } from '#preact';
 import {forwardRef} from '#preact/compat';
 
+import {installTimerService} from '#service/timer-impl';
+
 import {useStyles} from './component.jss';
 
 import {listen} from '../../../src/event-helper';
 import {Gestures} from '../../../src/gesture';
 import {SwipeXRecognizer} from '../../../src/gesture-recognizers';
+import {initLogConstructor} from '../../../src/log';
 
-const VALID_IMAGE_TAGNAMES = new Set(['AMP-IMG', 'IMG']);
+initLogConstructor();
+installTimerService(global);
 
 /**
  * Displays given component with supplied props.
  * @param {*} props
  * @param {{current: ?Element}} ref
+ * @param containerClass
  * @return {PreactDef.Renderable}
  */
-function DisplayAsWithRef({as: Comp = 'div', ...rest}, ref) {
+function DisplayAsWithRef({as: Comp = 'div', containerClass, ...rest}, ref) {
   return (
-    <div ref={ref}>
+    <div ref={ref} class={containerClass}>
```

Comment:
@deepaklalwani97 Would you please add comment describing why we need `containerClass` here?